meinberlin/apps/bplan/tasks.py

```diff
@@ -73,4 +73,4 @@ def get_location_information(bplan_id):
 
     topic = Topic.objects.get(code="URB")
     bplan.topics.set([topic])
-    bplan.save(update_fields=["point", "administrative_district", "topics"])
+    bplan.save(update_fields=["point", "administrative_district"])
```

Comment:
here just a `save()` is enough since the new fields are assigned in lines 64 and 65. And in line 75, should be `bplan.topics.add(topic)`

Reply:
just to clarify, that the `update_fields` get populated when we call `update()` rather than `save()`. Here for instance we could do `bplan.update(administrative_district = dis_dict[district_pk], point=point)` on line 64, instead of calling `save()` with the `updated_fields` arg. It skipped me last time I reviewed it.

Reply:
@m4ra thanks for the explanation, what do you suggest? stick with save or go for update()?

Reply:
@m4ra ok, this should have been documented but we do need the `update_fields`, otherwise it's an endless loop of celery tasks, see here: https://github.com/liqd/a4-meinberlin/blob/main/meinberlin/apps/bplan/signals.py#L11

Reply:
oh yes, `update()` bypasses `save()`, and updates directly the DB values.